src/components/Questionnaire/QuestionTypes/ConsentQuestion.tsx

```
@@ -0,0 +1,405 @@
+import useFileUpload from "@/hooks/useFileUpload";
+
+import { BACKEND_ALLOWED_EXTENSIONS } from "@/common/constants";
+
+import {
+  CONSENT_CATEGORIES,
+  CONSENT_DECISIONS,
+  CONSENT_STATUSES,
+  CreateConsentRequest,
+  VERIFICATION_TYPES,
+} from "@/types/consent/consent";
+import { FileUploadQuestion } from "@/types/files/files";
```

ESLint

[error] 'FileUploadQuestion' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)